code/kuusi/kuusi/settings.py

```diff
@@ -232,7 +256,12 @@
     "og:type": "website",
     "og:url": KUUSI_URL,
     "og:image": KUUSI_URL + "/logo.png",
+    "og:image:type": "image/png",
+    "og:image:width": "100",
+    "og:image:height": "100",
     "og:description": "",
+    "og:locale": "en_US",
```

Comment:
I think the `meta_tags` funciton in `web_extras` needs to overwrite this according to the currently selected language

Reply:
<img width="413" alt="image" src="https://github.com/user-attachments/assets/f2898bca-7e7c-4ebe-864e-5254a9808d24">

This modifies the function of setting the `og:locale` based on the current language.
?

Reply:
nvm - I've overseen that!